### DISCUSSION TITLE
Winget error upgrade command

### DISCUSSION THREAD

**Discussion by stopete** [microsoft/winget-cli#2787] [Q&A]
I keep getting the following error message when try to use winget ugprade trying researching the error but no luck  : "Failed when searching source: winget An unexpected error occurred while executing the command:  0x8a15000f : Data required by the source is missing" Any ideas.

**Comment by Trenly** [comment]
Take a look at this issue -
* https://github.com/microsoft/winget-pkgs/issues/86160
